docs/development.rst

```diff
@@ -93,58 +90,90 @@ To run migrations, you do::
 It'll perform any migrations that haven't been performed for all apps.
 
 
-Setting up an app for migrations
```

Comment:
Oh, no, that's fine. Just as long as there isn't some special `--initial` flag or other magic like South needed, we're good.